c-dependencies/js-compute-runtime/fastly.wit

```diff
@@ -423,6 +424,10 @@ default world fastly-world {
     // Should object store insert return "inserted" bool?
     object-store-insert: func(store: object-store-handle, key: string, body-handle: body-handle) -> result<_, error>
 
+    object-store-lookup-async: func(store: object-store-handle, key: string) -> result<pending-object-store-lookup-handle, error>
+
+    object-store-lookup-wait: func(h: pending-object-store-lookup-handle) -> result<option<body-handle>, error>
```

Comment:
Just wondering if there is a poll variation for this (I guess for `option<option<body-handle>>`)?

Reply:
We can use the fastly_async.is_ready hostcall, is that an equivalent of what you are describing?